manifests/common/neutron.pp

```diff
@@ -46,6 +46,7 @@
     enabled             => $is_controller,
     sync_db             => $is_controller,
     mysql_module        => '2.2',
+    router_distributed  => 'True',
```

Comment:
should be Boolean true-false

reference:
http://puppet-lint.com/checks/quoted_booleans/